services/libs/integrations/src/integrations/github/grid.ts

```diff
@@ -66,4 +66,8 @@ export const GITHUB_GRID: Record<GithubActivityType, IActivityScoringGrid> = {
     score: 6,
     isContribution: true,
   },
+  [GithubActivityType.AUTHORED_COMMIT]: {
+    score: 0, // not sure what to put here
```

Comment:
Can you check with Joan & Jonathan for author_commit score here? If we still want it 0, let's remove the comment